### ISSUE TITLE
Add language selector in the settings

### ISSUE THREAD

**Issue by BoFFire** [LucasGGamerM/moshidon#262]
Hi,

Is it possible to add an option in the settings to let the user select the UI language interface on Moshidon.

Some languages [or locales] such as `kabyle` code `kab` is not shown unless this kind of menu is shown to the user.

Thank you.

**Comment by LucasGGamerM** [comment]
Well, if you use Android 13 or later you can already individually set the application language from the phone's settings. I am not sure its feasible to implement such a feature to Moshidon at the moment

**Comment by Red1922** [comment]
> Well, if you use Android 13 or later you can already individually set the application language from the phone's settings. I am not sure its feasible to implement such a feature to Moshidon at the moment

I hope this is still considered in the future as Android does not support a few languages yet (be they uncommon or be they significant languages) and having the feature to change the UI language in the app itself would increase accessibility.
Additionally, I personally do not have an Android 13 device, nor does my OS support the language I wish to use and hence I have to rely on the app itself to provide translations.

**Comment by s6joui** [comment]
As someone who has a phone that will never get the Android 13 upgrade I would also appreciate this feature.

**Comment by LucasGGamerM** [comment]
Thank you for reminding me. I will take a look at this soon

**Comment by Ittihadyya** [comment]
Hi! is this issue still being considered?

**Comment by LucasGGamerM** [comment]
Yes (I have been a little slow lately, so that's why you haven't seen it already)

**Comment by nac0n** [comment]
I second this feature.
